gui/main.cxx

```diff
@@ -52,13 +56,42 @@ int main(int argc, char** argv)
   QApplication app(argc, argv);
   qtUtil::setApplicationIcon("mapgui");
 
+  bool help;
+  std::string openFilePath;
+
+  typedef kwiversys::CommandLineArguments argT;
+
+  argT arg;
+
+  arg.Initialize(argc, argv);
+
+  arg.AddBooleanArgument("--help", &help, "Display usage information");
+  arg.AddBooleanArgument("-h", &help, "Display usage information");
+  arg.AddArgument("--input", argT::SPACE_ARGUMENT,
+                  &openFilePath, "File to open at launch");
+  arg.AddArgument("-i", argT::SPACE_ARGUMENT,
```

Comment:
I think loading file from the command line is very helpful.  Thank you for contributing this.  However, I'm a bit undecided on the actual syntax of this.  All of the command line tools use `-c` to load in a config file.  So maybe this should be `-c`, `--config` as well for consistency?  On the other hand, the `openFile` method can support various other file formats as well.  So maybe it should remain `-i` but be an `argT::MULTI_ARGUMENT` so that you can load multiple files from the command line.  In theory that would allow something like this:

    mapgui -i frames/*.png tracks.txt output/landmarks.ply output/krtd/*.krtd

Of course that assumes everything is loaded in the right order.  Maybe both `-c` and `-i`?  Thoughts?  @mwoehlke-kitware , thoughts?